cvdupdate/cvdupdate.py

```diff
@@ -369,7 +369,14 @@ def _query_dns_txt_entry(self) -> bool:
         self.logger.debug(f"Checking available versions via DNS TXT entry query of current.cvd.clamav.net")
 
         try:
-            answer = str(resolver.resolve("current.cvd.clamav.net","TXT").response.answer[0])
+            our_resolver = resolver.Resolver()
+            nameserver = os.environ.get("CVDUPDATE_NAMESERVER")
+
+            if nameserver != None:
+                our_resolver.nameservers = [nameserver]
+                self.logger.info("Using nameserver specified in CVDUPDATE_NAMESERVER: {nameserver}")
```

Comment:
This needs to be an f-string to print the nameserver:
```python
                self.logger.info(f"Using nameserver specified in CVDUPDATE_NAMESERVER: {nameserver}")
```

Reply:
f-string added, documentation updated.